dali/core/mm/free_list_test.cc

```diff
@@ -148,6 +170,14 @@ TEST(MMFreeTree, PutGet) {
   TestCoalescingPutGet<free_tree>();
 }
 
+TEST(MMCoalescingFreeList, RemoveIf) {
+  TestCoalescingRemoveIf<coalescing_free_list>();
```

Comment:
Shouldn't you test `best_fit_free_list` as well?

Reply:
Well, that would require a different test. The implementation is almost trivial, though, and I don't plan to use it - it's there more or less for symmetry.

Reply:
Done.